flask_rebar/rebar.py

```diff
@@ -146,11 +146,19 @@ def wrapped(*args, **kwargs):
         if not response_body_schema:
             return rv
 
-        data, status_code, headers = _unpack_view_func_return_value(rv)
+        if isinstance(rv, current_app.response_class):
+            schema = response_body_schema[rv.status_code]
+            # The schema may be set to None to bypass marshaling (e.g. for 204 responses).
+            if schema is None:
+                return rv
+            # Otherwise, ensure the response body conforms to the promised schema.
+            schema.loads(rv.data)  # May raise ValidationError.
+            return rv
```

Comment:
I'm a bit concerned that the branching return here is an indicator that this code is in need of some refactoring. There schema validation assumes a dump were this validation requires a load, but I think otherwise we should be able to share one path. As it stands I'm worried about accidental drift. See normalize note.